app/assets/javascripts/search.js

```diff
@@ -1,114 +1,121 @@
+      search.trackSearchClicks($searchResults)
+      search.trackSearchResultsAndSuggestions($searchResults)
+      search.trackSearchResultsAndSuggestionsOnPageTrack($searchResults)
     },
     buildSearchResultsData: function ($searchResults) {
-      var searchResultData = {'urls': []},
-          searchURLs = search.extractSearchURLs($searchResults),
-          searchSuggestion = search.extractSearchSuggestion();
+      var searchResultData = { 'urls': [] }
+
+      var searchURLs = search.extractSearchURLs($searchResults)
+
+      var searchSuggestion = search.extractSearchSuggestion()
 
       if (searchURLs.length) {
-        searchResultData['urls'] = searchURLs.toArray();
+        searchResultData['urls'] = searchURLs.toArray()
       }
 
       if (searchSuggestion !== null) {
-        searchResultData['suggestion'] = searchSuggestion;
+        searchResultData['suggestion'] = searchSuggestion
       }
 
-      return searchResultData;
+      return searchResultData
     },
     enableLiveSearchCheckbox: function ($searchResults) {
-      GOVUK.liveSiteSearch.init();
+      GOVUK.liveSiteSearch.init()
     },
     extractSearchSuggestion: function () {
-      var $suggestion = $('.spelling-suggestion a');
+      var $suggestion = $('.spelling-suggestion a')
 
       if ($suggestion.length) {
-        return $suggestion.text();
+        return $suggestion.text()
       } else {
-        return null;
+        return null
       }
     },
     extractSearchURLs: function ($searchResults) {
       if ($searchResults.length <= 0) {
-        return [];
+        return []
       }
 
-      function extractSearchURL(index, element) {
-        var foundURL = $(element).find('h3 a');
+      function extractSearchURL (index, element) {
+        var foundURL = $(element).find('h3 a')
 
         if (foundURL.parents('.descoped-results').length) {
-          return $.makeArray(foundURL.map(function(index, item) {
+          return $.makeArray(foundURL.map(function (index, item) {
             return {
               href: $(item).attr('href'),
               descoped: true
-            };
-          }));
+            }
+          }))
         } else {
           return {
             href: foundURL.attr('href')
-          };
+          }
         }
       }
 
-      return $searchResults.children().map(extractSearchURL);
+      return $searchResults.children().map(extractSearchURL)
     },
     trackSearchClicks: function ($searchResults) {
-      if($searchResults.length === 0 || !GOVUK.cookie){
-        return;
+      if ($searchResults.length === 0 || !GOVUK.cookie) {
+        return
       }
 
-      $searchResults.on('click', 'a', function(e){
-        var $link = $(e.target),
-            sublink = '',
-            position, href, startAt;
+      $searchResults.on('click', 'a', function (e) {
+        var $link = $(e.target)
+
+        var sublink = ''
+
+        var position; var href; var startAt
```

Comment:
This is a bit inconsistent - two var declarations separated by newlines, then three all on one line. Can these be separated into individual lines, then remove the separating newlines?

Reply:
Absolutely! Frustrating standardjs doesn't handle this for us. I'll make the change 👍